plugins/fclrave/CMakeLists.txt

```diff
@@ -74,7 +80,7 @@ if( FCL_FOUND )
     if( CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR COMPILER_IS_CLANG)
       add_definitions("-std=c++11")
     endif()
-    set_target_properties(fclrave PROPERTIES COMPILE_FLAGS "${PLUGIN_COMPILE_FLAGS} ${FCL_CFLAGS_OTHER} -std=c++11" LINK_FLAGS "${PLUGIN_LINK_FLAGS} ${FCL_LDFLAGS}")
+    set_target_properties(fclrave PROPERTIES COMPILE_FLAGS "${PLUGIN_COMPILE_FLAGS} ${FCL_CFLAGS_OTHER} -std=c++11" LINK_FLAGS "${PLUGIN_LINK_FLAGS} ${openrave_FCL_LDFLAGS}")
```

Comment:
@cielavenir  `FCL_LDFLAGS`  should come from the compiled library's files. what is `openrave_FCL_LDFLAGS`?

Reply:
@cielavenir  just curious, what's the benefit of this change? was something breaking because `FCL_LDFLAGS` was improper?

Reply:
Let's move to https://github.com/rdiankov/openrave/pull/640#issuecomment-627095496